rust/kcl-lib/src/std/extrude.rs

```diff
@@ -121,16 +129,36 @@ async fn inner_extrude(
         ))
         .await?;
 
-        solids.push(do_post_extrude(sketch.clone(), id.into(), length, exec_state, args.clone()).await?);
+        solids.push(
+            do_post_extrude(
+                sketch.clone(),
+                id.into(),
+                length,
+                NamedCapTags {
+                    start: tag_start.clone(),
+                    end: tag_end.clone(),
+                },
+                exec_state,
+                args.clone(),
```

Comment:
None of these should clone. Same on all the `NamedCapTags`. Is there a reason you always clone, even when it's transferring ownership?

Reply:
im happy to change them to borrows but then ill just have to clone the inner values of NamedCapTags when i store them again

and args was already like that but i can fix, sketch too

Reply:
to be fair i fixed a bunch of random clones unrelated to the other pr here: https://github.com/KittyCAD/modeling-app/pull/5882/files#diff-eb6b385e26d238e79e97dc85a51fd0ac72786ec9c1b98d00b6d029ebddebed28R617 so hopefully im not the worst offender

Reply:
no i fixed them all because now i cant unsee!

Reply:
I also probably wrote some of that origiinal code i cant rememeber

Reply:
Yeah, I noticed the other PR changing to borrowed slices, and I like that!

It's not just a performance thing. It makes it a lot easier to read, in my opinion. Sometimes we need to clone for correctness, but if we clone everywhere, it's not clear why or if it can get deleted.